index.bs

```diff
@@ -936,27 +999,24 @@ To <dfn>create [=attribution aggregatable histograms=]</dfn> given an [=attribut
     1. [=list/Append=] |histogram| to |histograms|.
 1. Return |histograms|.
 
-<h3 algorithm id="triggering-attribution">Triggering attribution</h3>
+<h3 dfn id="is-there-sufficient-aggregatable-budget">Is there sufficient aggregatable budget</h3>
```

Comment:
Suggest naming this such that it is clear how the result is used. Maybe: "Can source generate aggregatable report"